docker/pre_commit_hook.sh

```diff
@@ -95,6 +95,9 @@ if [ "$is_container_running" != "0" ]; then
     make start-devserver
 fi
 
+# Add oppia as safe directory to git config
+$DOCKER_EXEC_COMMAND git config --global --add safe.directory /app/oppia
```

Comment:
Do we have to run it before every commit or can we do it once during setup?
Also do we have make a similar change for pre-push hook?

Reply:
Done!